config/kubernetes-sigs/sig-ui/teams.yaml

```diff
@@ -10,7 +10,19 @@ teams:
     description: Write access to the headlamp repo
     members:
     - joaquimrocha
+    - sniok
+    - illume
```

Comment:
Needs sorting the members list in alphabetical order here:

```
    - illume
    - joaquimrocha
    - sniok
```

Reply:
Completely forgot about this. Sorry. Will get it done now.